### ISSUE TITLE
Bank feed API

### ISSUE THREAD

**Issue by alelordelo** [sameersyd/Expenso#28]
Awesome project Sameer! Happy to contribute! : )

Has anyone added bank sync? Like Think or Plaid?
https://tink.com
https://plaid.com

That would make THE KILLER finance open source project!

**Comment by InsanelyHarsh** [comment]
Hello,
I would like to take this issue, can you explain more what is the task?

**Comment by alelordelo** [comment]
@InsanelyHarsh , currently the transactions are inputted manually. It would be awesome to have transactions synced from an API like Tink for example

**Comment by InsanelyHarsh** [comment]
okay, so I have to add transactions from tink to Expenso app, right?
Or sync both?

**Comment by alelordelo** [comment]
Yup,  the idea is to fetch all transactions from Tink, then you get a real feed from your bank 👍🙂

**Comment by InsanelyHarsh** [comment]
@alelordelo I am ready to take on this task!

**Comment by alelordelo** [comment]
Cool, looking forward to see the progress!
